components/docs-chef-io/content/habitat/service_restarts.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Service Restarts"
+description = "When does the supervisor restart a service"
+gh_repo = "habitat"
+
+[menu]
+  [menu.habitat]
+    title = "Service Restarts"
+    identifier = "habitat/reference/service-restarts Service Restarts"
+    parent = "habitat/reference"
++++
+
+Since the 1.6.477 Supervisor release, it is possible to configure a service restart backoff period to control how quickly a service is restarted following the
```

Comment:
Is this Habitat 1.6.477?

Reply:
Yeah that is actually the last release (the latest one as of right now). Maybe we should say since 1.6.491 which is the latest version in acceptance and it has this feature.